nix/README.md

```diff
@@ -128,8 +191,17 @@ sudo nixos-container start mina
 
 ### direnv
 
-It is considered as a good practice to automatically enter the Nix shell instead of keeping in mind that you need to execute the `nix develop mina` command every time you enter the Mina repo directory.  
-One way to do this is by using [`direnv`](https://direnv.net) + [`nix-direnv`](https://github.com/nix-community/nix-direnv)
+TL;DR
+```
+echo 'use flake mina' > .envrc
```

Comment:
I also have added a call to "./nix/pin.sh" prior to using the flake in my `.envrc` to avoid any synchronization issues, e.g.,  when I switch git branches with different flake configuration. Do you think it's worth referencing here, with a link towards the adequate section?